packages/blade/src/tokens/theme/createTheme.ts

```diff
@@ -0,0 +1,376 @@
+              lowContrast: foregroundOnBrandColorPaymentLight,
+            },
+          },
+        },
+        action: {
+          background: {
+            primary: {
+              default: brandColors[600],
+              hover: brandColors[700],
+              focus: brandColors[800],
+              active: brandColors[900],
+            },
+            secondary: {
+              default: brandColors.a00,
+              hover: brandColors.a50,
+              focus: brandColors.a100,
+              active: brandColors.a200,
+            },
+            tertiary: {
+              default: globalColors.neutral.blueGrayLight[0],
+              hover: globalColors.neutral.blueGrayLight[50],
+              focus: globalColors.neutral.blueGrayLight[100],
+              active: globalColors.neutral.blueGrayLight[200],
+            },
+          },
+          border: {
+            primary: {
+              default: brandColors[600],
+              hover: brandColors[700],
+              focus: brandColors[800],
+              active: brandColors[900],
+            },
+            secondary: {
+              default: brandColors[600],
+              hover: brandColors[600],
+              focus: brandColors[600],
+              active: brandColors[600],
+            },
+            tertiary: {
+              default: globalColors.neutral.blueGrayLight[300],
+              hover: globalColors.neutral.blueGrayLight[300],
+              focus: globalColors.neutral.blueGrayLight[300],
+              active: globalColors.neutral.blueGrayLight[300],
+            },
+          },
+          text: {
+            primary: {
+              default: foregroundOnBrandColorPaymentLight,
+              hover: foregroundOnBrandColorPaymentLight,
+              focus: foregroundOnBrandColorPaymentLight,
+              active: foregroundOnBrandColorPaymentLight,
+            },
+            secondary: {
+              default: foregroundOnSurfacePaymentLight,
+              hover: foregroundOnSurfacePaymentLight,
+              focus: foregroundOnSurfacePaymentLight,
+              active: foregroundOnSurfacePaymentLight,
+            },
+          },
+          icon: {
+            primary: {
+              default: foregroundOnBrandColorPaymentLight,
+              hover: foregroundOnBrandColorPaymentLight,
+              focus: foregroundOnBrandColorPaymentLight,
+              active: foregroundOnBrandColorPaymentLight,
+            },
+            secondary: {
+              default: foregroundOnSurfacePaymentLight,
+              hover: foregroundOnSurfacePaymentLight,
+              focus: foregroundOnSurfacePaymentLight,
+              active: foregroundOnSurfacePaymentLight,
+            },
+          },
+        },
+      },
+    },
+  };
+
+  return overrideTheme({
+    baseThemeTokens: paymentTheme,
+    overrides: lightThemePaymentsOverrides,
+  });
+};
+
+/**
+ *
+ * @param brandColors - The brand colors to use to override the banking theme
```

Comment:
why are we referring banking theme as dark theme? each themes has modes. payment light(light+dark) baking(light+dark) are we not following that structure here?

Reply:
So for dark theme I'm using the banking theme instead of payment's dark theme.